apptest/tests/vmauth_routing_test.go

```diff
@@ -301,3 +303,83 @@ unauthorized_user:
 	assertBackendsRequestsCount(1)
 
 }
+
+func TestSingleVMAuthUseProxyProtocol(t *testing.T) {
+	tc := apptest.NewTestCase(t)
+	defer tc.Stop()
+
+	var requestsCount int
+	var actualForwardedForHeader string
+	backend := httptest.NewServer(http.HandlerFunc(func(_ http.ResponseWriter, r *http.Request) {
+		actualForwardedForHeader = r.Header.Get("X-Forwarded-For")
+		requestsCount++
+	}))
+	defer backend.Close()
+
+	authConfig := fmt.Sprintf(`
+unauthorized_user:
+  url_prefix: %s
+  `, backend.URL)
+
+	vmauth := tc.MustStartVmauth("vmauth", []string{
+		"-httpListenAddr.useProxyProtocol=true",
+	}, authConfig)
+
+	makeGetRequestUsingProxyProtocol := func() {
```

Comment:
Why using a closure here?
Seems like it is only called once, so it is possible to remove the closure to run the testing directly.

Reply:
fixed in 1968f83926f2f046073d1fd456bfadaf9518987c